README.md

```diff
@@ -10,300 +10,95 @@
+-   [Roadmap](doc/roadmap.md)
 
-showConfigMigration
+### Development documentation
 
-	Shows the current migration configuration.
+-   [Development manual](doc/devel/development.md)
+-   [Contributing guide](doc/devel/contribution-guidelines.md)
+-   [Architecture](doc/devel/architecture.md)
+-   [North Port - NGSI Interactions](doc/devel/northboundinteractions.md)
 
-addProtocols <protocols>
+## IoT Agents available
 
-	Add a protocol translation table, in the following format:
-		protocolOrigin1=protocolTarget1;protocolOrigin2=protocolTarget2...
+The following IoT Agents using the **IoT Agent Node Lib** are available:
 
+-   [IoT Agent JSON](https://github.com/telefonicaid/iotagent-json) - a bridge between HTTP/MQTT messaging (with a JSON
+    payload) and NGSI
+-   [IoT Agent Ultralight](https://github.com/telefonicaid/iotagent-ul) - a bridge between HTTP/MQTT messaging
+-   [IoT Agent LWM2M](https://github.com/telefonicaid/lightweightm2m-iotagent) - a bridge between the
+    [Lightweight M2M](https://www.omaspecworks.org/what-is-oma-specworks/iot/lightweight-m2m-lwm2m/) protocol and NGSI
+    (with an UltraLight2.0 payload) and NGSI
+-   [IoT Agent for Sigfox](https://github.com/telefonicaid/sigfox-iotagent) - a bridge between the
+    [Sigfox](https://www.sigfox.com/en) protocol and NGSI
+-   [IoT Agent for LoRaWAN](https://github.com/Atos-Research-and-Innovation/IoTagent-LoRaWAN) - a bridge between the
+    [LoRaWAN](https://www.thethingsnetwork.org/docs/lorawan/) protocol and NGSI
+-   [IoT Agent for OPC-UA](https://github.com/Engineering-Research-and-Development/iotagent-opcua) - a bridge between
+    the [OPC Unified Architecture](http://www.opcua.us/) protocol and NGSI
+-   [IoT Agent for ISOXML](https://github.com/FIWARE/iotagent-isoxml) - a bridge between the ISOXML/ADAPT protocol for
+    agricultural machinery and NGSI
 
-migrate <targetDb> <service> <subservice>
+## Install and usage
 
-	Migrate all the devices and services for the selected service and subservice into the
-	specified Mongo database. To perform the migration for all the services or all the
-	subservices, use the "*" value.
-```
+The **IoT Agent node library** is not a standalone product. If you plan to install and use any of the IoT Agents
+available, you should follow the installation instructions for each IoT Agent (find the link in the
+[previous section](#iot-agents-available)). You can find the common API provided by the **IoT Agent node library** under
+[API](doc/api.md) documentation.
 
-The agent session stores transient configuration data about the target Context Broker and the target IoT Agent. This
-configuration is independent, and can be checked with the `showConfigCb` and `showConfigIot` commands, respectively.
-Their values can be changed with the `configCb` and `configIot` commands respectively. The new configurations will be
-deleted upon startup.
+If you plan to use the library in your own project or IoT Agent, you should follow the
```

Comment:
`IoT Agent node library` instead of just `library

Reply:
```suggestion
If you plan to use the IoT Agent node library in your own project or IoT Agent, you should follow the
```